src/scenes/Event/components/EventTxHistory/resultHistory.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
+import { FormattedMessage, injectIntl, intlShape } from 'react-intl';
+import Table, { TableBody, TableCell, TableHead, TableRow } from 'material-ui/Table';
+import Typography from 'material-ui/Typography';
+import { withStyles } from 'material-ui/styles';
+
+import { getLocalDateTimeString } from '../../../../helpers/utility';
+import styles from './styles';
+
+class EventResultHistory extends React.PureComponent {
+  render() {
+    const { classes, oracles } = this.props;
+
+    const sortedOracles = _.orderBy(oracles, ['endTime']);
+
+    return (
+      <div className={classes.detailTxWrapper}>
+        <Typography variant="headline" className={classes.detailTxTitle}>
+          <FormattedMessage id="str.resultHistory" defaultMessage="RESULT HISTORY" />
+        </Typography> {
+          sortedOracles.length ?
+            (<Table>
+              <TableHead>
+                <TableRow>
+                  <TableCell padding="dense">
+                    <FormattedMessage id="str.date" defaultMessage="Date" />
+                  </TableCell>
+                  <TableCell padding="dense">
+                    <FormattedMessage id="str.resultType" defaultMessage="Result Type" />
+                  </TableCell>
+                  <TableCell padding="dense">
+                    <FormattedMessage id="str.winningOutcome" defaultMessage="Winning Outcome" />
+                  </TableCell>
+                  <TableCell padding="dense">
+                    <FormattedMessage id="str.amount" defaultMessage="Amount" />
+                  </TableCell>
+                </TableRow>
+              </TableHead>
+              <TableBody>
+                {_.map(sortedOracles, (oracle, index) => (
+                  <TableRow key={`result-${index}`} selected={index % 2 === 1}>
+                    <TableCell padding="dense">{getLocalDateTimeString(oracle.endTime)}</TableCell>
+                    <TableCell padding="dense">{oracle.token === 'QTUM' ? 'QTUM Predict' : `Voting Round ${index}`}</TableCell>
```

Comment:
1. import constants for `Token.Qtum`
2. should we use `QTUM Betting` : `BOT Voting Round`?

Reply:
well bet/vote are actually actions, but here we are showing results, and we have the token name in the amount label already, so Betting Round : Voting Round?

Reply:
@liviavinci Betting Round and Voting Round sounds good :)